library/spdm_requester_lib/libspdm_req_challenge.c

```diff
@@ -321,6 +321,8 @@ return_status libspdm_challenge(IN void *context, IN uint8_t slot_id,
 	return_status status;
 
 	spdm_context = context;
+	spdm_context->req_timeout = spdm_context->connection_info.capability.rtt;
```

Comment:
This req_timeout isn't in the specification, but it would be the maximum amount of time the libspdm integrator can tolerate before a request is accepted by the Responder. It's not RTT.